cachito/web/migrations/versions/a655a299e967_request_flags.py

```diff
@@ -23,7 +23,7 @@ def upgrade():
         sa.Column("name", sa.String(), nullable=False),
         sa.Column("active", sa.Boolean(), nullable=False),
         sa.PrimaryKeyConstraint("id"),
-        sa.UniqueConstraint("id", "name"),
+        sa.UniqueConstraint("id", "name", name="flag_id_name_key"),
```

Comment:
I'm not 100% sure, but I think updates to old migration files will not be applied retroactively. Since `flag_id_name_key` is the default name, it should be fine to drop this change.

Reply:
I second this.

Reply:
Without the explicit name I get ValueError: No such constraint. How can I solve it without changing this file?

Reply:
It works for me. Here's how I tested this:
```shell
# start cachito on master branch
git checkout master
make clean
make run

# check current indexes
docker-compose exec db psql cachito -U cachito -c "select * from pg_indexes where tablename = 'flag'"

# checkout your branch
git checkout unique_flag_tbl
# also remove the explicit name from this file (only this file)

# test the upgrade and downgrade
docker-compose exec cachito-api cachito db upgrade
docker-compose exec db psql cachito -U cachito -c "select * from pg_indexes where tablename = 'flag'"
docker-compose exec cachito-api cachito db downgrade
docker-compose exec db psql cachito -U cachito -c "select * from pg_indexes where tablename = 'flag'"
```

Reply:
Oh I see, it's the unit tests that are failing, not the real migration. I'll take a look

Reply:
It appears that updating old migrations for this purpose is an accepted practice, I found an old commit which does exactly that: https://github.com/release-engineering/cachito/pull/184/commits/7b09691bec6b006a3f73c3f5cd633af961111513.

Let's keep this change, but please put it in a separate commit (and explain why it is needed in the commit message)